Examples/UIExplorer/ViewPagerAndroidExample.android.js

```diff
@@ -166,6 +168,9 @@ var ViewPagerAndroidExample = React.createClass({
           {pages}
         </ViewPagerAndroid>
         <View style={styles.buttons}>
+          <Button enabled={true} text={this.state.swipeEnabled ? 'Swipe Enabled': 'Swipe Disabled'} onPress={() => this.setState({swipeEnabled: !this.state.swipeEnabled})}/>
```

Comment:
Nit: Put the props on separate lines here so the line isn't too long and add a space before the `:` in the ternary expression.